spec/features/org_admin_cannot_be_added_to_new_org_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.feature "Org admin cannot be added to a new org" do
+  scenario "org admin fails to add an existing org admin" do
+    org_role = Role.create(name: 'org_admin')
+    reg_role = Role.create(name: 'registered_user')
+    reg_user = User.create(username: "bill@example.com", password: "password")
+    organization = Organization.create!(name:"Homes for Humanity", description: "We build homes", status: 1)
+    user = organization.users.create(username: 'fiona@cat.com', password: 'password')
+    user.roles << org_role
+    user.roles << reg_role
+    reg_user.roles << reg_role
+    reg_user.roles << org_role
```

Comment:
I think we want to remove line 11, and line 13 so that we can drive this behavior out in the test and not create a reg user with an org admin role in our set up, but do as you've done in line 24.